packages/protocol/contracts/common/libraries/SafeMathMem.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
```

Comment:
From what it looks like, if we care, we could [sublicense](https://softwareengineering.stackexchange.com/questions/105912/can-you-change-code-distributed-under-the-mit-license-and-re-distribute-it-unde) MIT code under GPL, for consistency with the rest of the contracts.

Speaking of which, I see a few edits here that add licensing info to some contracts, but there are still some without these comments. Worth it to create an issue to apply this to all contracts?